lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -1448,39 +1448,66 @@ void ApplyWorkingDir(SmallString &clang_argument, StringRef cur_working_dir) {
   llvm::sys::path::append(clang_argument, cur_working_dir, rel_path);
   llvm::sys::path::remove_dots(clang_argument);
 }
+
+std::array<StringRef> macro_flags = { "-D", "-U" };
+
+bool IsMultiArgClangFlag(const StringRef& arg) {
```

Comment:
Please use StringRef instead. StringRef only has a size of two pointers; it's designed to be passed by value.